scm/driver/harness/webhook.go

```diff
@@ -198,8 +223,8 @@ func convertPullRequestHook(dst *pullRequestHook) *scm.PullRequestHook {
 			Sha:    dst.Commit.Sha,
 			Ref:    dst.Ref.Name,
 			Author: scm.User{
-				Name:  dst.Commit.Committer.Identity.Name,
-				Email: dst.Commit.Committer.Identity.Email,
+				Name:  dst.PullReq.Author.DisplayName,
```

Comment:
there's more info we can set on scm user, like loginname among others?

(can we have common methods that convert `principal` to `scm.User` (same for others)